dashboard/src/features/accessControl/constants.js

```diff
@@ -20,3 +20,15 @@ export const policyOptions = [
     hint: 'Access to monitoring-specific endpoints'
   },
 ]
+
+export const subjectFieldOptions = [
```

Comment:
All-caps for `value` is a little more consistent to the way it will be used in the rest of the codebase.